webpack.config.js

```diff
@@ -16,7 +16,7 @@ let rules = [
     options: {
       presets: [
         [`@babel/preset-env`, { targets: `> 1%, last 2 versions` }],
-        [`@babel/preset-react`],
+        [`@babel/preset-react`, { runtime: "automatic" }],
```

Comment:
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#manual-babel-setup